### ISSUE TITLE
Summary=true serializer is slipping some things through - extensions on datatypes

### ISSUE THREAD

**Issue by brianpos** [ewoutkramer/fhir-net-api#583]
```
<Practitioner xmlns="http://hl7.org/fhir">
  <active value="true" />
  <name>
    <family value="Pos" />
    <given value="Brian" />
  </name>
  <telecom>
    <extension url="http://healthconnex.com.au/hcxd/Phone/IsMain">
      <valueBoolean value="true" />
    </extension>
    <system value="phone" />
    <value value="0417 144 099" />
  </telecom>
</Practitioner>
```
The extension on telecom should not come through in the summary serialization

**Comment by brianpos** [comment]
This is due to the code in the non generated file
https://github.com/ewoutkramer/fhir-net-api/blob/9c5f262cbab8200f0e82fffe3f0949a6e7120bc8/src/Hl7.Fhir.Core/Model/Element.cs#L72
Anyone know why we have marked the extension and ElementId in here as a part of the summary?

**Comment by ewoutkramer** [comment]
My guess is that initially, "extension" was marked as "in summary" when we generated it in September 2015. Then, we removed Element.cs from the list of generated classes, and this has remained so every since. So, I personally think we should make a manual correction here!

**Comment by ewoutkramer** [comment]
Fixed by #635
